code/modules/mob/floating_message.dm

```diff
@@ -19,8 +19,8 @@ var/global/list/floating_chat_colors = list()
 	var/fontsize = 6
 	if(small)
 		fontsize = 5
-	var/limit = 50
-	if(copytext(message, length(message) - 1) == "!!")
+	var/limit = 130
+	if(copytext(message, length(message) - 1) in list("!!","??"))
```

Comment:
`??` seems like a bit of a odd choice for confused shouting. Would `?!`/`!?` make more sense?